lib/process-services-cloud/src/lib/models/process-services-cloud-query.model.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pagination } from '@alfresco/js-api';
+
+export interface ProcessServicesCloudResponse {
```

Comment:
Not sure ProcessServicesCloudResponse is the right name, should be more something related to user preference? like UserPreferenceCloudModel

Reply:
the response is the same for all cloud services, its not just for the user preferences. for example, all cloud query services return the same structure and it is mapped with "any" type. We could use this model across cloud query services. if "ProcessServicesCloudResponse" name is not sounds good, we can find a better name.

Reply:
the response is the same for all cloud services, its not just for the user preferences. for example, all cloud query services return the same structure and it is mapped with "any" type. We could use this model across cloud query services. if "ProcessServicesCloudResponse" name does not sound good, we can find a better name.